modules/microshift-config-parameters-table.adoc

```diff
@@ -139,14 +160,44 @@ Not all MIME types benefit from compression, but `HAProxy` uses resources to try
 |`443`
 |Default port shown. Configurable. Valid value is a single, unique port in the `1-65535` range. The values of the `ports.http` and `ports.https` fields cannot be the same.
 
-|`ingress.routeAdmissionPolicy.namespaceOwnership`
+//Has this parameter been renamed to 'ingress.routeAdmission'?
+
+|`ingress.routeAdmissionPolicy`
 |`Strict` or `InterNamespaceAllowed`
-|Describes how hostname claims across namespaces are handled. By default, allows routes to claim different paths of the same hostname across namespaces. Specifying `Strict` prevents routes in different namespaces from claiming the same hostname. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
+|Defines a policy for handling new route claims, such as allowing or denying claims across namespaces. By default, allows routes to claim different paths of the same hostname across namespaces. Specifying `Strict` prevents routes in different namespaces from claiming the same hostname. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
+
+|`ingress.routeAdmissionPolicy.wildcardPolicy`
+|`WildcardsAllowed`, `WildcardsDisallowed`
+|Describes how routes with wildcard policies are handled by the Ingress Controller.
+
+* `WildcardsAllowed`: Indicates routes with any wildcard policy are admitted by the Ingress Controller.
+
+* `WildcardsDisallowed`: Indicates only routes with a wildcard policy of `None` are admitted by the Ingress Controller. Updating `wildcardPolicy` from `WildcardsAllowed` to `WildcardsDisallowed` causes admitted routes with a wildcard policy of `Subdomain` to stop working. These routes must be recreated to a wildcard policy of `None` to be readmitted by the Ingress Controller. `WildcardsDisallowed` is the default setting.
 
 |`ingress.status`
 |`Managed` or `Removed`
 |Router status. Default is `Managed`.
 
+|`ingress.tlsSecurityProfile`
+|`object`
+|Specifies settings for TLS connections for Ingress Controllers. If you do not set one, the default value is based on the `apiservers.config.openshift.io/cluster` resource.
+
+|`ingress.tlsSecurityProfile.type`
+|`Old`, `Intermediate`, `Modern`, 'Custom' 
+|Specifies the profile type for the TLS Security.
+
+When using the `Old`, `Intermediate`, and `Modern` profile types, the effective profile configuration is subject to change between releases. For example, given a specification to use the `Intermediate` profile deployed on release `X.Y.Z`, an upgrade to release `X.Y.Z+1` may cause a new profile configuration to be applied to the Ingress Controller, resulting in a rollout.
```

Comment:
`Intermediate` is the current default (when nothing is provided), i think we should make this more clear.